### PR TITLE
[AttentionClipper] Doc Modified and [split fore background] add conversion for depth image format 32FC1

### PR DESCRIPTION
**Pull request by knorth55**
[Attention Clipper] add more information for doc 
[split fore background] add conversion for depth image format 32FC1

### CODE REVIEW

File: doc/jsk_pcl_ros/nodes/attention_clipper.md [hunk 1]
```diff
@@ -28,10 +28,12 @@ to see the object.
 * `~output` (`sensor_msgs/CameraInfo`)
 
   This camera info is same with `~input` except for roi field.
+  (only when `~input` is subscribed.)
```
Comment (wkentaro) [inline doc/jsk_pcl_ros/nodes/attention_clipper.md]: only when `~input` comes?

File: doc/jsk_pcl_ros/nodes/attention_clipper.md [hunk 2]
```diff
@@ -28,10 +28,12 @@ to see the object.
...
 
 * `~output/mask` (`sensor_msgs/Image`)
 
   Mask image to mask the regions of specified interest.
+  (only when `~input` is subscribed.)
```
Comment (wkentaro) [inline doc/jsk_pcl_ros/nodes/attention_clipper.md]: only when `~input` comes?

File: jsk_perception/node_scripts/split_fore_background.py [hunk 3]
```diff
@@ -30,16 +32,21 @@ def unsubscribe(self):
 
     def _apply(self, img_msg, depth_msg):
         # validation
-        if depth_msg.encoding != '16UC1':
+        if depth_msg.encoding == '16UC1':
```
Comment (wkentaro) [inline jsk_perception/node_scripts/split_fore_background.py]: Oops, this line should not be modified.

**Comment by wkentaro** [comment]
Also, please change the doc for split_fore_background.py.

**Comment by wkentaro** [comment]
#1345 will fix both of them.

**Comment by knorth55** [comment]
thank you

**Pull request merged**